src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public static String getRandomBall() {
```

Comment:
method should not be static
```suggestion
    public String getRandomBall() {
```

Reply:
done